src/Grb.Building.Api/Handlers/GetActiveJobsHandler.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Grb.Building.Api.Handlers
+{
+    using System;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Infrastructure;
+    using MediatR;
+    using Microsoft.EntityFrameworkCore;
+    using TicketingService.Abstractions;
+
+    public sealed record GetActiveJobsRequest() : IRequest<GetActiveJobsResponse>;
+
+    public sealed record GetActiveJobsResponse(JobResult[] Jobs);
+
+    public sealed class GetActiveJobsHandler
+        : IRequestHandler<GetActiveJobsRequest, GetActiveJobsResponse>
+    {
+        private readonly BuildingGrbContext _buildingGrbContext;
+        private readonly ITicketingUrl _ticketingUrl;
+        private readonly IPagedUriGenerator _pagedUriGenerator;
+
+        public GetActiveJobsHandler(
+            BuildingGrbContext buildingGrbContext,
+            ITicketingUrl ticketingUrl,
+            IPagedUriGenerator pagedUriGenerator)
+        {
+            _buildingGrbContext = buildingGrbContext;
+            _ticketingUrl = ticketingUrl;
+            _pagedUriGenerator = pagedUriGenerator;
+        }
+
+        public async Task<GetActiveJobsResponse> Handle(
+            GetActiveJobsRequest request,
+            CancellationToken cancellationToken)
+        {
+            var result = await _buildingGrbContext
+                .Jobs
+                .Where(x => x.Status != JobStatus.Cancelled && x.Status != JobStatus.Completed)
+                .ToListAsync(cancellationToken);
+
+            return new GetActiveJobsResponse(
+                result.Select(x =>
+                    new JobResult
```

Comment:
I don't like the class name JobResult as this is a concept with another meaning.